react/TextField/TextField.js

```diff
@@ -72,20 +74,22 @@ export default class TextField extends Component {
     this.handleMouseDownOnClear = this.handleMouseDownOnClear.bind(this);
   }
 
+  storeContainerReference = textField => {
+    if (textField !== null) {
+      this.container = textField;
+    }
+  }
+
   storeInputReference(input) {
     if (input !== null) {
       this.input = input;
     }
   }
 
   handleMouseDownOnClear(event) {
-    const { onClear } = this.props;
-
-    if (typeof onClear === 'function') {
```

Comment:
because I used invoke instead, so there's technically no need in it...only if for some reason we don't want to prevent focus too